### ISSUE TITLE
LocalVideoTrack.stop() or LocalAudioTrack.stop() causes screen to freeze on Chrome

### ISSUE THREAD

**Issue by gritbranch** [twilio/twilio-video.js#1288]
- [x] I have verified that the issue occurs with the latest twilio-video.js release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-video.js/blob/master/CHANGELOG.md).
 - [x] I reviewed the [Common Issues](https://github.com/twilio/twilio-video.js/blob/master/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
 - [x] I verified that the [Quickstart](https://github.com/twilio/video-quickstart-js) application works in my environment.
 - [x] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.

Hi there,

We use twilio-video with Angular and we are encountering an issue wherein the UI freezes when using LocalAudioTrack or LocalVideoTrack stop() method. This only happens only to Chrome browser on an Android 11. I have done some testing on Chrome on Android 9 and Chrome Android 10 and they all work fine.

**Code to reproduce the issue:**

```js
// TODO
get tracks() {
  return this.localTracksPromise;
}

localTracksPromise: Promise<LocalTrack[]>;

ngOnInit() { 
	...
	this.localTracksPromise = createLocalTracks({
	  audio: true,
	  video: {
			// Video parameters here...
	  }
	});
}


leave() {
	...
	this.tracks.then(tracks => {
		tracks.forEach(track => {
		  if (track.kind === 'audio' || track.kind === 'video') {
		    track.stop();
		    track.detach().forEach(element => {
		      try {
		        if (element) {
		          element.remove();
		        }
		      } catch (e) {
		        console.error(e);
		      }
		    });
		  }
		});
	});

	this.router.navigate(['/login']).then();	
}
```

**Expected behavior:**

Video and audio tracks are stopped and the app navigates away from the video call.

**Actual behavior:**

UI freezes and the browser does not respond to back button or pull to refresh . Only way is to turn screen off / on then the browser will refresh.

I also tried commenting the `track.stop();` and the app did not freeze. 

TODO

**Software versions:**

 - [x] Browser(s): Chrome 86.0.4240.198
 - [x] Operating System: Android 11
 - [x] twilio-video.js: twilio-video v.2.7.0
 - [x] Third-party libraries (e.g., Angular, React, etc.): Angular v10.1.5

**Comment by makarandp0** [comment]
Hey @gritbranch - This is [known issue](https://bugs.chromium.org/p/chromium/issues/detail?id=1138823) with chrome on Android 11. We are not aware of any workaround short of avoiding track.stop, But will keep you updated as we know more about it. 

Thanks,
Makarand

**Comment by gritbranch** [comment]
Thank you Makarand. I'll keep an eye on the link you posted and will be monitoring from there. Cheers.

**Comment by gavinlefebvre** [comment]
@gritbranch try swapping the order of your track.stop() and the track.detach() block- the root cause seems to be a rendering issue that is no longer an issue if the track has already stopped.

**Comment by gritbranch** [comment]
Thank you for the suggestion. I have tried swapping the order of the track.stop() and track.detach() but unfortunately we are still encountering the issue with Chrome on Android 11.
